quarkdown-html/src/main/scss/components/_code.scss

```
@@ -70,6 +70,35 @@
     color: white;
   }
 
+  // Code block callouts
+
+  .code-callout-marker {
+    @include font.global-font-family();
```

Stylelint (17.14.0)

[error] Unexpected parentheses in argumentless mixin "font.global-font-family" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)